larq/quantizers_test.py

```diff
@@ -48,78 +48,112 @@ def test_invalid_usage():
         lq.quantizers.get("unknown")
 
 
-@pytest.mark.parametrize("name", ["ste_sign", "approx_sign", "swish_sign"])
-def test_binarization(name):
+def generate_real_values_with_zeros(low=-2, high=2, shape=(2, 5)):
```

Comment:
If you want to be fancy, you can use a [pytest.fixture](https://docs.pytest.org/en/latest/fixture.html) for this.